### DISCUSSION TITLE
PKHeX for Analogue Pocket?

### DISCUSSION THREAD

**Discussion by CodebyCR** [kwsch/PKHeX#4367] [Q&A]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when if I need to start my PC only to modify a save file.

**Describe the solution you'd like**
A very nice solution were to the availibility to modify the save file directly on my `Analogue Pocket`.
I think many Pokémon enthusiast play on the Analogue Pocket and the Analogue Pocket cores save the progress directly in a .sav file. This would be fit very well in the ecosystem.
A modified subset version of pkhex for the Analoge Pocket which can modify .sav files of the first three Pokémon Generations souves this gap.

**Describe alternatives you've considered**
I'm unsure whether there any real alternatives maybe use a online sav file modifier and your phone conneced to the Analogue Pocket, but this is also a unfinish souluion in my opinion.

**Comment by Lusamine** [comment]
PKHeX is a PC program. [PKHeX.Mobile](https://github.com/kwsch/PKHeX.Mobile) was attempted a long time ago but very few wanted to contribute to it, so it's largely fallen out of use.

The program is open source. You or anyone else is welcome to try making it run on anything you like. Having to maintain the program for even more niche systems is not a goal of this repo.

**Comment by kwsch** [comment]
PKHeX is a Windows Forms application, which means it was written as a PC program for the GUI.

If you want something that runs on specialized hardware (like a game console), which has an entirely different graphics API, then you need a different application/codebase.

**Comment by codemonkey85** [comment]
I know it's been a while, but if you're looking for something you can use quickly on the go, I've made a web app powered by PKHeX: [PKMDS: Pokémon Save Editor for Web](https://pkmds.app/). There are others as well, such as https://pkhex-web.github.io/.

**Reply by eddyramgb** [reply]
Habrá alguna posibilidad de que PKMDS se pueda volver APK para android?

Is there any possibility that PKMDS could be turned into an APK for Android?

**Reply by codemonkey85** [reply]
No, but you can install it as a PWA, and after the initial load it should work offline, although sprites may not work correctly.

No, pero puedes instalarla como PWA y, tras la carga inicial, debería funcionar sin conexión, aunque es posible que los *sprites* no funcionen correctamente.

**Reply by eddyramgb** [reply]
I tried it, but it stops working when I close the internet connection.

**Reply by codemonkey85** [reply]
I just tested, and you can still load, edit, and export a save file while offline on Android. ~~However, I did notice a bit of a delay in loading the file. I'll look into that.~~ EDIT: never mind, that's only from loading the file from Google Drive. Loading a file directly from the local device storage is normal.

**Reply by sofianeelhor** [reply]
https://github.com/sofianeelhor/PKForge
